common/include/Pcsx2Defs.h

```diff
@@ -121,64 +113,68 @@ static const int __pagesize = PCSX2_PAGESIZE;
-//#	define __pack_end		__pragma(pack())
+	// Using these breaks compat with VC2005; so we're not using it yet.
+	//#	define __pack_begin		__pragma(pack(1))
+	//#	define __pack_end		__pragma(pack())
 
-// This is the 2005/earlier compatible packing define, which must be used in conjunction
-// with #ifdef _MSC_VER/#pragma pack() directives (ugly).
-#define __packed
+	// This is the 2005/earlier compatible packing define, which must be used in conjunction
+	// with #ifdef _MSC_VER/#pragma pack() directives (ugly).
+	#define __packed
 
-#define __aligned(alig) __declspec(align(alig))
-#define __aligned16 __declspec(align(16))
-#define __aligned32 __declspec(align(32))
-#define __pagealigned __declspec(align(PCSX2_PAGESIZE))
+	#define __aligned(alig) __declspec(align(alig))
+	#define __aligned16 __declspec(align(16))
+	#define __aligned32 __declspec(align(32))
+	#define __pagealigned __declspec(align(PCSX2_PAGESIZE))
 
-#define __noinline __declspec(noinline)
+	#define __noinline __declspec(noinline)
 
-// Don't know if there are Visual C++ equivalents of these.
-#define likely(x) (!!(x))
-#define unlikely(x) (!!(x))
+	// Don't know if there are Visual C++ equivalents of these.
+	#define likely(x) (!!(x))
+	#define unlikely(x) (!!(x))
 
-#define CALLBACK __stdcall
+	#define CALLBACK __stdcall
 
 #else
 
 // --------------------------------------------------------------------------------------
 //  GCC / Intel Compilers Section
 // --------------------------------------------------------------------------------------
 
-#ifndef __packed
-#define __packed __attribute__((packed))
-#endif
-#ifndef __aligned
-#define __aligned(alig) __attribute__((aligned(alig)))
-#endif
-#define __aligned16 __attribute__((aligned(16)))
-#define __aligned32 __attribute__((aligned(32)))
-#define __pagealigned __attribute__((aligned(PCSX2_PAGESIZE)))
-
-#define __assume(cond) ((void)0) // GCC has no equivalent for __assume
-#define CALLBACK __attribute__((stdcall))
-
-// Inlining note: GCC needs ((unused)) attributes defined on inlined functions to suppress
-// warnings when a static inlined function isn't used in the scope of a single file (which
-// happens *by design* like all the friggen time >_<)
-
-#ifndef __fastcall
-#define __fastcall __attribute__((fastcall))
-#endif
-#define _inline __inline__ __attribute__((unused))
-#ifdef NDEBUG
-#define __forceinline __attribute__((always_inline, unused))
-#else
-#define __forceinline __attribute__((unused))
-#endif
-#ifndef __noinline
-#define __noinline __attribute__((noinline))
-#endif
-#define likely(x) __builtin_expect(!!(x), 1)
-#define unlikely(x) __builtin_expect(!!(x), 0)
+	#ifndef __packed
+		#define __packed __attribute__((packed))
+	#endif
+	#ifndef __aligned
+		#define __aligned(alig) __attribute__((aligned(alig)))
+	#endif
+	#define __aligned16 __attribute__((aligned(16)))
+	#define __aligned32 __attribute__((aligned(32)))
+	#define __pagealigned __attribute__((aligned(PCSX2_PAGESIZE)))
```

Comment:
While _aligned16 and _pagealigned are used, _aligned32 isn't actually used anywhere in the code that I see. _aligned(32) and _aligned(64) are used a few times...

Reply:
I replaced them all with `alignas` since we have C++11 now

Reply:
Sounds good to me.